src/NServiceBus.RabbitMQ/Delays/DelayInfrastructure.cs

```diff
@@ -0,0 +1,111 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Text;
+    using global::RabbitMQ.Client;
+
+    static class DelayInfrastructure
+    {
+        const int maxNumberOfBitsToUse = 28;
+        const int maxLevel = maxNumberOfBitsToUse - 1;
+        public const int MaxDelayInSeconds = (1 << maxNumberOfBitsToUse) - 1;
```

Comment:
I agree. I must have overlooked that, because that's something I typically do automatically.